include/databroker/configuration/DatabrokerConfiguration.hpp

```diff
@@ -50,7 +50,7 @@ class DatabrokerConfiguration
 
 protected:
 
-    std::list<AbstractTopic*> common_topic_list_get_(
+    std::list<std::shared_ptr<AbstractTopic>> common_topic_list_get_(
```

Comment:
Also I see that there is a `try catch` in its implementation but I don't see where the exception could be raised.

Reply:
1. :+1: 
2. In this method the YAML is parsed, I thought it would be better to have it with try catch.
I think if the list element in the YAML is bad formed (dictionary instead of array or viceversa) it could generate and exception, but I am not sure

Reply:
Fair enough